### PR TITLE
Add use of logging instead of print statements

### PR DESCRIPTION
**Pull request by kriation**
I had difficulty troubleshooting an issue I was experiencing using this utility, so I added logging, to be able to debug not only this code, but the use of its dependent libraries.

This is not feature complete, as I'm sure there is room for improvement but it provided the necessary functionality for me to successful diagnose and resolve my issue.

### CODE REVIEW

**Comment by kriation** [comment]
I'll work on adding tests for the additional argument over the weekend.

**Comment by brainstorm** [comment]
Same here, I totally endorse this PR, can you please merge @cevoaustralia?

**Comment by kriation** [comment]
Hi @stevemac007, could you review and merge this PR? [@brainstorm](https://github.com/cevoaustralia/aws-google-auth/pull/127#issuecomment-474620661) and I think this would be useful to add to the code base.

File: aws_google_auth/__init__.py [hunk 1]
```diff
@@ -66,7 +71,7 @@ def cli(cli_args):
         config = resolve_config(args)
         process_auth(args, config)
     except google.ExpectedGoogleException as ex:
-        print(ex)
+        logging.exception(ex)
```
Comment (stevemac007) [inline aws_google_auth/__init__.py]: section should be:
```
    except google.ExpectedGoogleException as ex:
        print(ex)
        sys.exit(1)
    except KeyboardInterrupt:
        pass
    except Exception as ex:
        logging.exception(ex)
```
The ExpectedGoogleException is used for application flow, and so should eat the exception trace - see scenarios where user supplies invalid password.

Comment (kriation) [reply aws_google_auth/__init__.py]: Thanks for the explanation @stevemac007 ! I appreciate it.

**Comment by kriation** [comment]
> Looks good - and works well locally for me.
> I would like the error messages cleaned up - but I'll merge and fix this.
> 
> Thanks for the contribution - and sorry on the radio silence.
My pleasure! Thank you for merging it into the code base.

**Pull request merged**